esrally/utils/convert.py

```diff
@@ -25,6 +25,10 @@ def bytes_to_human_string(b):
     return "%d bytes" % b
 
 
+def number_of_documents_to_human_string(number_of_documents):
```

Comment:
I think this method name is too special as it can be used to format any number. Can you please rename it to `number_to_human_string`?

Reply:
Fixed it